spec.html

```diff
@@ -35,10 +35,10 @@ <h1>Syntax</h1>
       PipelineExpression[In, Yield, Await] :
         LogicalORExpression[?In, ?Yield, ?Await]
         [~Await] PipelineExpression[?In, ?Yield, ?Await] `|>` LogicalORExpression[?In, ?Yield, ?Await]
-        [+Await] PipelineExpression[?In, ?Yield, ?Await] `|>` [lookahead &lt;! {`await`}] LogicalORExpression[?In, ?Yield, ?Await]
-        [+Await] PipelineExpression[?In, ?Yield, ?Await] `|>` `await` LogicalORExpression[?In, ?Yield, ?Await]
     </ins>
   </emu-grammar>
+
+  It is a Syntax Error if `await` appears in the `LogicalORExpression` the RHS of a `PipelineExpression`.
```

Comment:
If you keep the lookahead restriction, it would ban literal await following a pipeline. With that, I don't see why we'd need a syntax error.

Nit: syntax errors tend to be presented in Static Semantics: Early Error blocks.

Reply:
This is _kinda_ what I thought but I wasn't sure. I'll remove this, since the wording isn't quite right.